### PR TITLE
Pretty output !goals command

### LINKED ISSUE #46: !goals command output is not alligned
Find solution to fix alignment of columns in the output of !goals

### PR DESCRIPTION
**Pull request by CSSilverIV**
Fixes #46

### CODE REVIEW

File: playerstats.py [hunk 1]
```diff
@@ -36,13 +36,15 @@ def getPlayerStats(club):
...
+    df_topscorers.index = df_topscorers.index + 1
+    table = tabulate(df_topscorers, tablefmt='plain', colalign=['left', 'left'], showindex=False)
     return table
 
+
 def main(msgLower):
     match msgLower:
-      case '!goals':
-          return getGoalsScored('Arsenal')
+        case '!goals':
```
Comment (ash663) [inline playerstats.py:49]: I would suggest not to change the tab spaces

Comment (CSSilverIV) [reply playerstats.py:49]: I didn't. The second commit was to restore the tab space, which was changed by mistake

File: playerstats.py [hunk 2]
```diff
@@ -36,13 +36,15 @@ def getPlayerStats(club):
     df_topscorers = pd.DataFrame.from_dict(top_scorer)
     return df_topscorers
 
+
 def getGoalsScored(club):
     df_topscorers = getPlayerStats(club).sort_values(by=['goals'], ascending=False, ignore_index=True).head(5)
-    df_topscorers.index = df_topscorers.index+1
-    table=tabulate(df_topscorers,tablefmt='plain',colalign=['left','left'],showindex=False)
+    df_topscorers.index = df_topscorers.index + 1
```
Comment (ash663) [inline playerstats.py:42]: This can be removed if we're not showing indices

Comment (CSSilverIV) [reply playerstats.py:42]: Cool. Let it be for now. I'll remove it in the !assist commit

**Review by ash663 — APPROVED** [review]
Just added a small non blocking np comment. Lgtm

**Review by ingenium21 — APPROVED** [review]

**Pull request merged**